coalib/bears/Bear.py

```diff
@@ -100,6 +130,30 @@ def supported_languages(cls):
         return (cls.LANGUAGES if isinstance(
             cls.LANGUAGES, tuple) else (cls.LANGUAGES,))
 
+    @classproperty
+    def maintainers(cls):
+        """
+        :return: A tuple with maintainers names.
+        """
+        return ((cls.MAINTAINERS,) if isinstance(
+            cls.MAINTAINERS, str) else cls.MAINTAINERS)
+
+    @classproperty
+    def maintainers_emails(cls):
+        """
+        :return: A tuple with maintainers emails.
+        """
+        return ((cls.MAINTAINERS_EMAILS,) if isinstance(
+            cls.MAINTAINERS, str) else cls.MAINTAINERS_EMAILS)
+
+    @classproperty
+    def include_local_files(cls):
+        """
+        :return: A tuple with local file paths to be included.
+        """
+        return ((cls.INCLUDE_LOCAL_FILES,) if isinstance(
+            cls.INCLUDE_LOCAL_FILES, str) else cls.INCLUDE_LOCAL_FILES)
```

Comment:
oh yes for those you'll need tests as they're functions, which aren't evaluated on module load right?